### DISCUSSION TITLE
调度监控

### DISCUSSION THREAD

**Discussion by user757187977** [apache/dolphinscheduler#7359] [Q&A]
我想制作调度器的大盘，监控调度的核心指标，期望有人一起沟通下，关于调度的 SLA，以及有木有已经实现了调度器的大盘或者指标监控。
希望大家给我些建议，调度的 SLA 都有哪些？以前调度的 SLA 都需要哪些指标支撑。

**Comment by davidzollo** [comment]
you can contact my through mail(lidongdai@apache.org or wechat: taskflow), when connected, please tell me your github id
